controller/controller_test.go

```diff
@@ -1565,7 +1564,7 @@ cmd:
 			Expect(err).NotTo(HaveOccurred())
 			scheduler := models.NewScheduler(configYaml1.Name, configYaml1.Game, yamlWithLimit)
 
-			mt.MockScaleUp(mockPipeline, mockRedisClient, configYaml1.Name, configYaml1.AutoScaling.Max - currentRooms)
```

Comment:
Was this change suggested by linter?
I usually like to keep empty spaces in calculations like that to make it easier to visualize.

Reply:
this is gofmt.